### PR TITLE
Add yaml to hack folder to enable mesh-wide mTLS

### PR DESCRIPTION
**Pull request by xeviknal**
** Describe the change **
Adding a hack yaml which enables meshwide mTLS to the whole Istio installation.

### CODE REVIEW

File: hack/istio-mtls-meshwide.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+##########
+# This enables mTLS for the whole istio installation.
```
Comment (abonas) [inline hack/istio-mtls-meshwide.yaml]: istio -> Istio

Comment (abonas) [reply hack/istio-mtls-meshwide.yaml]: also, it's not for Istio itself, right? Istio installation might be misleading

Comment (xeviknal) [reply hack/istio-mtls-meshwide.yaml]: Comment approached, phrase reformulated :)

**Comment by jmazzitelli** [comment]
Please put this yaml file in the "istio" directory under "hack" with the rest of them ... we have a bunch of these istio yaml files there - keep them all together. The "hack" directory is mainly for scripts starting and managing the k8s/OS cluster.

**Comment by xeviknal** [comment]
@jmazzitelli moved. You are right. I was mistaken when I created that file.

**Pull request merged**